pkg/querier/queryrange/codec.go

```diff
@@ -633,6 +688,25 @@ func toProto(m loghttp.Matrix) []queryrange.SampleStream {
 	return res
 }
 
+func toProtoVector(v loghttp.Vector) []queryrange.SampleStream {
```

Comment:
I wonder whether we could add `ToProto` to `loghttp.Vector`. I tried but the package has not access to to `queryrange`. It's a little inconsistent because `loghttp.Streams` does have `ToProto` defined.